src/utils/rnasum.js

```diff
@@ -0,0 +1,252 @@
+    samples_no: '168',
+  },
+  {
+    project: 'PAAD',
+    name: 'Pancreatic Adenocarcinoma',
+    tissue_code: '1',
+    samples_no: '150',
+  },
+  {
+    project: 'TGCT',
+    name: 'Testicular Germ Cell Tumours',
+    tissue_code: '1',
+    samples_no: '149',
+  },
+  {
+    project: 'LAML',
+    name: 'Acute Myeloid Leukaemia',
+    tissue_code: '3',
+    samples_no: '145',
+  },
+  {
+    project: 'ESCA',
+    name: 'Esophageal Carcinoma',
+    tissue_code: '1',
+    samples_no: '142',
+  },
+  {
+    project: 'GBM',
+    name: 'Glioblastoma Multiforme',
+    tissue_code: '1',
+    samples_no: '141',
+  },
+  {
+    project: 'THYM',
+    name: 'Thymoma',
+    tissue_code: '1',
+    samples_no: '118',
+  },
+  {
+    project: 'SKCM',
+    name: 'Skin Cutaneous Melanoma',
+    tissue_code: '1',
+    samples_no: '100',
+  },
+  {
+    project: 'READ',
+    name: 'Rectum Adenocarcinoma',
+    tissue_code: '1',
+    samples_no: '87',
+  },
+  {
+    project: 'UVM',
+    name: 'Uveal Melanoma',
+    tissue_code: '1',
+    samples_no: '80',
+  },
+  {
+    project: 'ACC',
+    name: 'Adrenocortical Carcinoma',
+    tissue_code: '1',
+    samples_no: '78',
+  },
+  {
+    project: 'MESO',
+    name: 'Mesothelioma',
+    tissue_code: '1',
+    samples_no: '77',
+  },
+  {
+    project: 'KICH',
+    name: 'Kidney Chromophobe',
+    tissue_code: '1',
+    samples_no: '59',
+  },
+  {
+    project: 'UCS',
+    name: 'Uterine Carcinosarcoma',
+    tissue_code: '1',
+    samples_no: '56',
+  },
+  {
+    project: 'DLBC',
+    name: 'Lymphoid Neoplasm Diffuse Large B-cell Lymphoma',
+    tissue_code: '1',
+    samples_no: '47',
+  },
+  {
+    project: 'CHOL',
+    name: 'Cholangiocarcinoma',
+    tissue_code: '1',
+    samples_no: '34',
+  },
+];
+
+export const EXTENDED_DATASETS_OPTION = [
+  {
+    no: '1',
+    project: 'LUAD-LCNEC',
+    name: 'Lung Adenocarcinoma dataset including large-cell neuroendocrine carcinoma (LCNEC, n=14)',
+    tissue_code: '1',
+    samples_no: '314',
+  },
+  {
+    no: '2',
+    project: 'BLCA-NET',
+    name: 'Bladder Urothelial Carcinoma dataset including neuroendocrine tumours (NETs, n=2)',
+    tissue_code: '1',
+    samples_no: '248',
+  },
+  {
+    no: '3',
+    project: 'PAAD-IPMN',
+    name: 'Pancreatic Adenocarcinoma dataset including intraductal papillary mucinous neoplasm (IPMNs, n=2)',
+    tissue_code: '1',
+    samples_no: '152',
+  },
+  {
+    no: '4',
+    project: 'PAAD-NET',
+    name: 'Pancreatic Adenocarcinoma dataset including neuroendocrine tumours (NETs, n=8)',
+    tissue_code: '1',
+    samples_no: '158',
+  },
+  {
+    no: '5',
+    project: 'PAAD-ACC',
+    name: 'Pancreatic Adenocarcinoma dataset including acinar cell carcinoma (ACCs, n=1)',
+    tissue_code: '1',
+    samples_no: '151',
+  },
+];
+
+export const PAN_CANCER_DATASETS_OPTION = [
+  {
+    no: '1',
+    project: 'PANCAN',
+    name: 'Samples from all [33 cancer types](#primary-datasets), 10 samples from each',
```

Comment:
I'd probably get rid of the markdown link here ;-)

Reply:
Whoops! Thanks for checking this out 😅